Pipfile

```diff
@@ -6,7 +6,7 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
-elyra = "==2.0.1"
+elyra = "==2.2.1"
```

Comment:
It needs 

```
[pipenv]
allow_prereleases = true
```

because of `black` package for Airflow. (not sure why black is mandatory for airflow direct dependencies, as it seems listed in `devel` https://github.com/apache/airflow/blob/aebacd74058d01cfecaf913c04c0dbc50bb188ea/setup.py#L480) @akchinSTC

Reply:
nice catch @pacospace :clap: 
@akchinSTC which version of `pipenv` are you using?

Reply:
@pacospace - thanks for the fix, updating now!
@harshad16 - pipenv, version 2020.11.15, let me know if I need to use a different version and regen the piplock

Reply:
thank you , this seems to be the latest version
wondering via it didn't autoupdate the allow_prereleases in the Pipfile.